### ISSUE TITLE
See Owned Tokens on Hidden Layer

### ISSUE THREAD

**Issue by aliasmask** [RPTools/maptool#1761]
**Is your feature request related to a problem? Please describe.**
It would be nice to have the option for an owned token to be seen, but not able to be clicked on and I believe putting an owned token on the Hidden Layer could accomplish this. This would be different than putting it on the object layer which allows the token to be seen through soft FoW.

**Describe the solution you'd like**
Render owned tokens on the Hidden layer. They shouldn't expose FoW or be clicked on, but can be seen by players if they own the token.

**Describe alternatives you've considered**
This seems like the simplest solution to a typical invisible token that can be seen by some of the players. It disallows players from interacting with the token and treats it like a token layer token for vision. ie blocked by vbl and FoW when viewed from another selected token also owned by player. One thing though is that it shouldn't show up with shared vision (no tokens selected). Purely individual views only. Since it can never be selected by the player, it can only be seen when exposed and has line of sight to it from another token's perspective.

** Summary
Owned token on Hidden Layer
* Visible only from another owned token with individual views or seen by any PC token with group view.
* Can't be selected by player.
* Can't be seen through VBL, FoW or soft FoW.

Maybe adding a new checkbox and functions to toggle checkbox would easier to implement. Call it Expose Owned Hidden Tokens. This also sound somewhat related to another request regarding exposing owned tokens above FoW.

**Comment by aliasmask** [comment]
I've come up with a better idea. So, scrap this one.
